### ISSUE TITLE
browserify error without babel

### ISSUE THREAD

**Issue by nichoth** [chec/commerce.js#164]
```
Error: Can't walk dependency graph: Cannot find module '@babel/runtime/helpers/defineProperty' from 'node_modules/@chec/commerce.js/lib/index.js'
```

You can clone and see https://github.com/nichoth/commerce-demo/tree/d5d79975dc083c35d56becf33d317824e10621cb

**Comment by robbieaverill** [comment]
Hi @nichoth, what are you doing with that example repository to trigger this error?

**Comment by robbieaverill** [comment]
Steps to reproduce:

* Clone repository: https://github.com/nichoth/commerce-demo/tree/d5d79975dc083c35d56becf33d317824e10621cb
* `npm install`
* `npm run build`

Terminal output:
```
$ npm run start

> start
> npm run build && concurrently --kill-others "npm run serve" "npm run sass-watch"


> build
> npm run copy && npm run sass && npm run build-js


> copy
> mkdir -p public && mkdir -p public/uploads && mkdir -p public/admin && cp src/admin/* public/admin && cp src/index.html public && mkdirp -p src/uploads && cp src/uploads/* public/uploads


> sass
> sass ./src/style/style.scss ./public/style.css


> build-js
> browserify --debug -p esmify src/index.js > public/bundle.js

Error: Can't walk dependency graph: Cannot find module '@babel/runtime/helpers/typeof' from 'node_modules/@chec/commerce.js/lib/index.js'
    required by /Users/abc123/commerce-demo/node_modules/@chec/commerce.js/lib/index.js
    at /Users/abc123/commerce-demo/node_modules/esmify/esmify.js:47:19
    at /Users/abc123/commerce-demo/node_modules/browser-resolve/index.js:265:24
    at /Users/abc123/commerce-demo/node_modules/resolve/lib/async.js:139:17
    at processDirs (/Users/abc123/commerce-demo/node_modules/resolve/lib/async.js:290:39)
    at isdir (/Users/abc123/commerce-demo/node_modules/resolve/lib/async.js:297:32)
    at /Users/abc123/commerce-demo/node_modules/resolve/lib/async.js:25:69
    at FSReqCallback.oncomplete (fs.js:168:21)
```

**Comment by robbieaverill** [comment]
@nichoth thanks for raising this issue. We've released v2.5.3, which contains the missing Babel dependency.

I've also made a PR to your repository to update Commerce.js for you: https://github.com/nichoth/commerce-demo/pull/1
